src/device-registry/utils/does-component-exist.js

```diff
@@ -48,6 +48,47 @@ const doesDeviceExist = async (deviceName, tenant) => {
   }
 };
 
+const filterExistingDevices = (measurements) => {
+  let promises = measurements.filter(async (measurement) => {
+    try {
+      logText(".......................................");
```

Comment:
In the near future, we might actually not need these utils, all we have to do is call the list devices method of the Device entity and proceed accordingly based on the result/response we get. 
More like: 
```

 const responseFromListDevice = await getModelByTenant(
        tenant,
        "device",
        DeviceSchema
      ).list({ name:deviceName})

if (responseFromListDevice.success===false){
// respond to the bad response/error accordingly
}
else {
// go ahead and perform winning actions, all is good...
}
```